tests/components/knx/test_config_flow.py

```diff
@@ -958,6 +958,7 @@ async def test_options_flow_connection_type(
     hass.data[DOMAIN] = Mock()  # GatewayScanner uses running XKNX() instance
     gateway = _gateway_descriptor("192.168.0.1", 3675)
 
+    await hass.config_entries.async_setup(mock_config_entry.entry_id)
```

Comment:
Do we need to patch the library to avoid I/O? If we don't need the integration actually setup we can alternatively patch out `async_setup_entry` in the init module.